src/main/java/frc/robot/Drive.java

```diff
@@ -23,14 +24,18 @@ public Drive(SwerveDrivetrain drivetrain, DoubleSupplier fwdSupplier, DoubleSupp
         this.rtSupplier = rtSupplier;
     }
 
+    @Override
```

Comment:
So this inverts the Joy input when we are RED. Will this not create a situation in field centric mode since the robot is started facing the Blue side?

Reply:
this works because gyro get flipped according to team color in the FieldLocation enum